updateVersions.sc

```diff
@@ -1,6 +1,6 @@
-val workbenchUtil2 = "0.10-c3ef6b80-SNAP"
+// Must be run with scala 2.13 or above.
 
-interp.configureCompiler(_.settings.YpartialUnification.value = true)
```

Comment:
This line causes the script to fail if you run with scala 2.13. The feature is now enabled by default. https://stackoverflow.com/questions/61035986/value-ypartialunification-is-not-a-member-of-scala-tools-nsc-settings